src/frontend/components/UI/Anticheat/index.scss

```diff
@@ -6,6 +6,7 @@
   border-radius: 20px;
   background-color: var(--current-background);
   margin-bottom: var(--space-lg);
+  margin-top: var(--space-lg);
```

Comment:
can you check if it is affecting the install dialog as well? since it uses the same component?

Reply:
install dialog is looking fine